types/google-apps-script/google-apps-script.document.d.ts

```diff
@@ -1462,6 +1465,53 @@ declare namespace GoogleAppsScript {
      *     table.getCell(0, 2).setVerticalAlignment(DocumentApp.VerticalAlignment.BOTTOM);
      */
     enum VerticalAlignment { BOTTOM, CENTER, TOP }
+
+    interface Date extends Element {
```

Comment:
Move between `ContainerElement` and `Document` (sort alphabetically).
Person, RickLink also.

Reply:
Done.

Reply:
also added some comments.